lib/3scale_toolbox/tasks/copy_activedocs_task.rb

```diff
@@ -0,0 +1,24 @@
+module ThreeScaleToolbox
+  module Tasks
+    class CopyActiveDocsTask
+      include CopyTask
+
+      def call
+        puts 'copying all service ActiveDocs'
+
+        target_activedocs = source.list_activedocs.map do |source_activedoc|
+          source_activedoc.clone.tap do |target_activedoc|
+            target_activedoc.delete('id')
+            target_activedoc['service_id'] = target.id
+            target_activedoc['system_name'] = "#{target_activedoc['system_name']}#{target.id}"
+          end
+        end
+
+        target_activedocs.each do |ad|
+          res = target.remote.create_activedocs(ad)
+          raise Error, "ActiveDocs has not been created. Errors: #{res['errors']}" unless res['errors'].nil?
```

Comment:
It is hard to know which class the Error resolves to and would be nicer to use the full namespace.

Reply:
updated to use full namespace